deployment/src/main/java/io/quarkiverse/wiremock/devservice/WireMockServerProcessor.java

```diff
@@ -54,8 +56,15 @@ DevServicesResultBuildItem setup(LaunchModeBuildItem launchMode, LiveReloadBuild
     private static RunningDevService startWireMockDevService(WireMockServerConfig config) {
 
         LOGGER.debugf("Starting WireMock server with port [%s] and path [%s]", config.port(), config.filesMapping());
-        final WireMockServer server = new WireMockServer(
-                options().port(config.port()).usingFilesUnderDirectory(config.filesMapping()));
+        final WireMockConfiguration configuration = options()
+                .port(config.port())
+                .usingFilesUnderDirectory(config.filesMapping());
+
+        if (config.globalResponseTemplating()) {
+            configuration.extensions(new ResponseTemplateTransformer(true));
+        }
```

Comment:
It looks like this is the way how you would do it in `WireMock 2`. With `3.X` the API has been enhanced: https://wiremock.org/3.x/docs/response-templating/. 

Furthermore, `WireMock 3` enables `Response templating` by default, but you need to add a `response-template` transformer for each stub. To avoid this, you can enable it globally via
```
WireMockServer wm =
    new WireMockServer(options().globalTemplating(true));

```

Reply:
Obsolete